documentation/modules/operators/con-operators-namespaces.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// assembly-operators.adoc
+
+[id='con-operators-namespaces-{context}']
+
+= Watching namespaces with Strimzi operators
+
+[role="_abstract"]
+Operators watch and manage Strimzi resources in namespaces.
+The Cluster Operator can watch a single namespace, multiple namespaces, or all namespaces in a Kubernetes cluster.
+The Topic Operator and User Operator can watch a single namespace.
+
+* The Cluster Operator watches for `Kafka` resources
+* The Topic Operator watches for `KafkaTopic` resources
+* The User Operator watches for `KafkaUser` resources 
+
+The Topic Operator and the User Operator can only watch a single Kafka cluster in a namespace.
+And they can only be connected to a single Kafka cluster.  
+
+If a Topic Operator watches more than one namespace, name collisions and topic deletion can occur. This is because each Kafka cluster uses Kafka topics that have the same name (such as `__consumer_offsets`). 
```

Comment:
```suggestion
If multiple Topic Operators watch the same namespace, name collisions and topic deletion can occur.
This is because each Kafka cluster uses Kafka topics that have the same name (such as `__consumer_offsets`).
You should make sure that only one Topic Operator watches given namespace.
```